addons/link_tracker/controller/main.py

```diff
@@ -5,15 +5,23 @@
 from odoo.http import request
 
 
+ignore_user_agent_prefix = ["twitterbot", "facebookexternalhit", "facebot",
+                            "LinkedInBot", "WhatsApp", "Googlebot", "Pinterest", "Instagram"]
```

Comment:
Your commit message can be improved
```
Add a manual list of user agent prefixes in odoo/addons/link_tracker/controller/main.py , then add an if statement on the route.
```

Here you describe what you did in your code, but I can see it by opening the commit.

A grand sage would say; the commit should not be really about "What", but more about "Why": "The social media need to build a preview, for that they make an HTTP call to the URLs in the posts. Those HTTP requests are currently count as user click". And "How": "Those social media have a specific user agent in the HTTP headers that can be used to detect them and to not increment the click count in that case".

Also update your PR message :)